src/App.tsx

```diff
@@ -1,29 +1,61 @@
 import React, { Component } from 'react';
-// @ts-ignore
+import {BrowserRouter as Router, Route, Link } from 'react-router-dom';
 import logo from './logo.svg';
 import './App.css';
 
-class App extends Component {
+class AppRouter extends Component {
+  render() {
+    return (
+      <Router>
+        <div>
+          <Route path='/' exact component={HomePage} />
+          <Route path='/login' component={LoginPage} />
+        </div>
+      </Router>
+    );
+  }
+}
+
+interface User {
+  email: string;
+}
+
+class HomePage extends Component {
+  readonly state: {user: User | null} = {user: null};
+
+  componentDidMount() {
+    this.fetchUser();
+  }
+
+  fetchUser() {
+    fetch('/current-user')
+      .then((response) => response.json())
+      .then((data: {user: User}) => {
+        this.setState({user: data.user});
+      });
+  }
+
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
+          <h1>Kvasir Movies</h1>
           <p>
-            Edit <code>client-src/App.js</code> and save to reload.
+            {this.state.user != null ? `Hi ${this.state.user.email}! ` : ''}
+            Find 🎬 with 👫 :D
```

Comment:
Not blocking: we should make this text bigger so we can see the emojis better!

Reply:
Oh, yeah this is bs UI, I hope we throw all this away